src/Infrastructure/Persistence/Configuration/CategoryConfiguration.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Codidact.Core.Domain.Entities;
```

Comment:
Since these files are generated you should add the following to `.gitattributes`:

~~~none
/src/Infrastructure/Persistence/Configuration/* linguist-generated
~~~